build-tools/packages/build-tools/src/common/npmPackage.ts

```diff
@@ -319,6 +319,13 @@ export class Packages {
         return this.queueExecOnAllPackage(pkg => pkg.noHoistInstall(repoRoot), "npm i");
     }
 
+    public async filteredPackages(releaseGroup: MonoRepoKind | undefined) {
```

Comment:
Nit: Since this is a method, should this be `filterPackages` rather than `filteredPackages`?

Reply:
Good catch. It started as a property -- will change.